C23_WMDE_Desktop_DE_24

Sperling-0 opened the pull request:
- Based on Var banner of 22
- Add direct redirect feature
- VAR banner has text change
- The amount that's missing is formatted as x.x00.000 Euro
- Adjust the tests accordingly
- update 'content' version

Ticket: https://phabricator.wikimedia.org/T353637

Abban commented on desktop_preact/styles/FundsModal.pcss:
```diff
@@ -0,0 +1,201 @@
+		background-color: #b8e986;
+	}
+	&__org--wmfg {
+		background-color: #f8e71c;
```
Should be `#fcf48e`

Abban commented on desktop_preact/styles/FundsModal.pcss:
```diff
@@ -0,0 +1,201 @@
+		display: inline-block;
+	}
+	&__org--wmf {
+		background-color: #b8e986;
```
Should be `#dcf4c3`

Abban commented on desktop_preact/styles/FundsModal.pcss:
```diff
@@ -0,0 +1,201 @@
+		background-color: #f8e71c;
+	}
+	&__org--wmde {
+		background-color: #f5a623;
```
Should be `#fbd392`

Abban commented on desktop_preact/components/FundsModal.jsx:
```diff
@@ -0,0 +1,149 @@
+							<div className="banner_modal__benefits_list">
+								<h2>Was Ihre Spende bewirkt</h2>
+								<ul className="banner_modal__icon-list">
+									<li className='banner_modal__icon-list_item--hand'>
```
Thinking about the video we watched yesterday, regarding redundant words in class and method names. BEM sorta does the same thing in markup eg `<li class="list-item"></li>`

Abban commented on desktop_preact/styles/FundsModal.pcss:
```diff
@@ -0,0 +1,201 @@
+	&__button {
+		padding: 15px;
+		/* stylelint-disable-next-line declaration-property-value-blacklist */
+		border: none;
```
`border: 0;` will work here and not upset the linter.

gbirke commented on desktop_preact/styles/FundsModal.pcss:
```diff
@@ -0,0 +1,201 @@
+		padding: 0;
+
+		li {
+			margin: 0 0 14px 55px;
```
The outer margin (30px) comes from `.banner_modal__container`, so it's 60px. If UX decides that the grey areas should stretach to the sides I'll adjust it.

gbirke commented:
Thank you for all the hints! I disagree with the color changes - I've lifted the colors directly form the Figma document. I'd rather try to export the image as SVG and see if they are the same then.

gbirke commented:
Work continued in #397

Sperling-0 committed:
Prepare campaign for C23_WMDE_Desktop_DE_24

- Based on the VAR of test 22
- Add direct redirect feature

Ticket: https://phabricator.wikimedia.org/T353637

Sperling-0 committed:
Implement VAR for C23_WMDE_Desktop_DE_24

- VAR banner has text change
- Add direct redirect feature
- The amount that's missing is formatted as x.x00.000 Euro
- Adjust the tests accordingly
- update 'content' version
- Remove campaign overrides for development environment

Ticket: https://phabricator.wikimedia.org/T353637

Abban committed:
Add currentDonationSum to dynamic text

As part of the urgency text we need to add a formatted version
of the current donation sum to the content.

Abban commented on campaign_info.toml:
```diff
@@ -17,13 +17,13 @@ use_of_funds_source = "MediaWiki:WMDE_Fundraising/UseOfFunds_2023_DE"
-pagename = "B23_WMDE_Desktop_DE_22_ctrl"
-tracking = "org-22-231218-ctrl"
+pagename = "B23_WMDE_Desktop_DE_24_ctrl"
+tracking = "org-22-231224-ctrl"
```
The tracking needs to be updated to 24 eg: `org-24-231224-ctrl`

Sperling-0 replied:
Done.
